_posts/_2021-10-05-Launch-Announcement-1-1.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+authors: 
+  - henkle
+  - ahopp
+comments: true
+title: "OpenSearch 1.1 is here!"
+category:
+- releases
+---
+
+To the OpenSearch community everywhere, 
+
+We are excited to announce the 1.1 release of the OpenSearch project (available to [download](https://opensearch.org/downloads.html) today). Not only are we excited that this release marks the first minor version release on the OpenSearch project, it also comes with a number of new and exciting features. And, best of all, we continue to see the community grow and contribute to OpenSearch!
+
+So, what’s in 1.1? This release includes dozens of new features and hundreds of improvements (Release Notes) but we wanted to highlight some of the most exciting new features in this release;
+
+- Cross Cluster Replication: This is one we are personally really excited about. Being able to deploy OpenSearch clusters across different servers, data centers, or even regions will help anyone setup a more fault-tolerant configuration. Further, you can now store different indices on different clusters and query data across all of them  with a single request.
+- Lucene 8.9 Upgrade: OpenSearch 1.1 supports Lucene 8.9!  Highlights include sorting improvements when building BKD indexes that can boost index building by 35x and speed up  merging sorted fields and term vectors for smaller segments. It also includes a fix for a performance regression that was introduced in 8.8 for Boolean Queries. Full details are available in the Lucene [release notes](https://lucene.apache.org/core/8_9_0/changes/Changes.html). 
+- CLI OpenSearch Upgrade Tool: While upgrading to OpenSearch from a compatible version of Elasticsearch, there are some manual tasks that need to be performed. The opensearch-upgrade tool included in the distribution automates those tasks by importing the existing configurations and applying them to the new installation of OpenSearch. This is the first version of the tool, so we welcome your feedback for future enhancements! For more information see the [readme](). We’re also planning a blogpost that highlights how to best use this feature. 
```

Comment:
is opensearch-upgrade the name of the tool on the command line? if so, that should be in back ticks.

Reply:
Done.